VideoLocker/src/main/java/org/edx/mobile/view/NavigationFragment.java

```diff
@@ -95,10 +93,7 @@ public void onCreate(Bundle savedInstanceState) {
             getAccountTask = new GetAccountTask(getActivity(), profile.username) {
                 @Override
                 protected void onSuccess(@NonNull Account account) throws Exception {
-                    NavigationFragment.this.profileImage = account.getProfileImage();
-                    if (null != imageView) {
-                        loadProfileImage(account.getProfileImage(), imageView);
-                    }
+                    // Image loading logic can be found in onEventMainThread(AccountDataLoadedEvent)
```

Comment:
If we're not going to override any callback method, then the base `GetAccountTask` class should be declared non-abstract and an instance of it should simply be instantiated, instead of having to define an unneeded anonymous subclass. The comment can be moved back to the initialization logic.

The comment can also be changed to include the reasoning, e.g.:

> The data will be posted on the global event bus as a AccountDataLoadedEvent, so we're listening for the event and handling it from there.